src/test/scala/org/ergoplatform/nodeView/state/UtxoStateSpecification.scala

```diff
@@ -94,17 +94,18 @@ class UtxoStateSpecification extends ErgoPropertyTest with ErgoTransactionGenera
       )
       val unsignedTx = new UnsignedErgoTransaction(inputs, IndexedSeq(), newBoxes)
       val tx = ErgoTransaction(defaultProver.sign(unsignedTx, IndexedSeq(foundersBox), emptyDataBoxes, us.stateContext).get)
-      val validationRes1 = us.validateWithCost(tx, 100000)
+      val validationContext = us.stateContext.simplifiedUpcoming()
+      val validationRes1 = us.validateWithCost(tx, Some(validationContext), 100000, None)
```

Comment:
Looks like `validateWithCost` is called everywhere with `Some(validationContext)`. 
Thus, us.stateContext.simplifiedUpcoming() probably can be called from validateWithCost.